modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3233,9 +3251,9 @@ If a query ID is marked as unknown, it means either the query does not exist or
 
 `GET /restpp/query_result`
 
-This endpoint allows you to check the results of queries run in Detached Mode if they have finished running. If the query is still running, the endpoint will respond with an error and a message saying `"Unable to retrieve result for query <requestid>"`.  Ensure that the query is finished before checking its result.
+This endpoint allows you to check the results of queries run in Detached Mode if they have finished running. 
```

Comment:
1. Use similar wording as in the other modified endpoints.
2. Why did you cut out the sentences "If the query is still running..."?